src/client/styles.css

```diff
@@ -14,6 +14,19 @@
   text-decoration: none;
 }
 
+.scene-list-item:hover, .caption-list-item:hover {
+  background: #cccccc;
```

Comment:
I'd like to have any hex code be explained in a quick comment.

Reply:
Done!